packages/css/table.css

```diff
@@ -1,143 +1,110 @@
-  background-color: var(--dsc-table-header-sorted-background);
-}
+  & > :is(tbody, thead) > tr > :is(th, td) {
+    background: var(--dsc-table-background);
+    border-bottom: 1px solid var(--dsc-table-border-color);
+    padding: var(--dsc-table-padding);
+    text-align: inherit;
+    &:is(th) { font-weight: 500 }
+  }
 
-.ds-table__header__cell--sortable button:focus-visible {
-  z-index: 3;
-  outline-offset: -3px;
-  box-shadow: unset;
-}
+  & > thead > tr > :is(th, td) {
+    border-bottom: 2px solid var(--dsc-table-border-color);
+    background: var(--dsc-table-header-background);
+  }
 
-.ds-table__header__cell--sortable button svg {
-  font-size: 1.2em;
-}
+  /* Add rounded border to first and last row */
+  & > :is(thead, tbody):first-child > tr:first-child > :is(th, td) {
+    &:first-child { border-top-left-radius: var(--dsc-table-border-radius) }
+    &:last-child { border-top-right-radius: var(--dsc-table-border-radius) }
+  }
 
-.ds-table__cell {
-  padding: var(--dsc-table-padding);
-  border-bottom: 1px solid var(--dsc-table-border-color);
-  background-color: var(--dsc-table-cell-backround);
-}
+  /* Add rounded border to last row */
+  & > :is(thead, tbody):last-child > tr:last-child > :is(th, td) {
+    &:first-child { border-bottom-left-radius: var(--dsc-table-border-radius) }
+    &:last-child { border-bottom-right-radius: var(--dsc-table-border-radius) }
+  }
 
-.ds-table--zebra .ds-table__row {
-  border-bottom: 0;
-}
+  /**
+    * Sorting
+    */
+  & > thead > tr > [aria-sort] {
+    cursor: pointer;
+
+    & > button {
+      all: unset;
+      display: inline-block;
+      vertical-align: middle;
+      &:focus-visible { box-shadow: var(--dsc-focus-boxShadow) }
+    }
+
+    &::after {
+      background: currentcolor;
+      content: '';
+      display: inline-block;
+      height: var(--dsc-table-sort-size);
+      mask: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Cpath d='M12.53 4.47a.75.75 0 0 0-1.06 0l-3.5 3.5a.75.75 0 0 0 1.06 1.06L12 6.06l2.97 2.97a.75.75 0 1 0 1.06-1.06zm-3.5 10.5a.75.75 0 0 0-1.06 1.06l3.5 3.5a.75.75 0 0 0 1.06 0l3.5-3.5a.75.75 0 1 0-1.06-1.06L12 17.94z'/%3E%3C/svg%3E");
+      vertical-align: middle;
+      width: var(--dsc-table-sort-size);
+    }
+    &[aria-sort="ascending"]::after { mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Cpath d='M11.47 7.97a.75.75 0 0 1 1.06 0l5.5 5.5a.75.75 0 1 1-1.06 1.06L12 9.56l-4.97 4.97a.75.75 0 0 1-1.06-1.06z'/%3E%3C/svg%3E") }
+    &[aria-sort="descending"]::after { mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Cpath d='M5.97 9.47a.75.75 0 0 1 1.06 0L12 14.44l4.97-4.97a.75.75 0 1 1 1.06 1.06l-5.5 5.5a.75.75 0 0 1-1.06 0l-5.5-5.5a.75.75 0 0 1 0-1.06'/%3E%3C/svg%3E") }
+    &:not([aria-sort="none"]) { background: var(--dsc-table-header-background--sorted) }
+  }
```

Comment:
Any reason for why you use `mask` for the "none" icon, and `mask-image` for the two others?

Reply:
Good catch! ❤️  I see I forgot the rest of the props in the `mask`-shorthand- will fix ☺️